src/Microsoft.Azure.SignalR.Common/ServiceMessages/RuntimeServicePingMessages.cs

```diff
@@ -14,7 +16,12 @@ internal static class RuntimeServicePingMessage
         private const string StatusKey = "status";
         private const string ShutdownKey = "shutdown";
         private const string ServersKey = "servers";
-
+        private const string DiagnosticLogsKey = "diag";
+        private const string DiagnosticLogsMessagingTypeKey = "MessagingLogs";
+
+
+        private const string MessagingLogEnableValue = "1";
```

Comment:
these are part of protocol, why can't they go to Microsoft.Azure.SignalR.Protocol?

Reply:
let's fix it in a seperate PR.